lib/modules/core/chabok.js

```diff
@@ -11,19 +11,31 @@ export const playServicesAvailability = AdpNativeModule.playServicesAvailability
 
 export class AdpPushClient {
 
-    initializeApp = (appName, options, cbk) => {
-        AdpNativeModule.initializeApp(appName, options, response => {
-            cbk(response);
-        });
+    initializeApp = (options) => {
+        if (!options){

```

Comment:
check options object keys

Reply:
You are reviewing after approval !? How is this approved then?

Reply:
After my discussion with @Husseinhj we decided to hold the PR for extra changes and after that reviewing it.

Reply:
👍